scripts/internal/workflow-helpers/jira.go

```diff
@@ -101,3 +102,15 @@ func IsMergedStatus(status string) bool {
 	}
 	return false
 }
+
+func FetchJiraIDsInCommits(commits []*github.RepositoryCommit) []string {
```

Comment:
It doesn't look like this function is used anywhere except in a test. Is there any reason to keep it?

Reply:
Yes, to help me debug issues if/when they occur :p